raiden/network/transport.py

```diff
@@ -30,11 +28,18 @@ class TokenBucket(object):
     """Implementation of the token bucket throttling algorithm.
     """
 
-    def __init__(self, capacity=10., fill_rate=10.):
+    def __init__(self, capacity=10., fill_rate=10., time=None):
         self.capacity = float(capacity)
         self.fill_rate = fill_rate
         self.tokens = float(capacity)
-        self.timestamp = time.time()
+
+        if time is None:
+            from time import time
```

Comment:
Only importing inside the function to avoid name collisions. I'm afraid that we can't have a standard for all scenarios because sometimes a import must be delayed due to python dynamic nature... nevertheless this is not one of this cases, so, I'm just going to rename the variables and import at the top.